### DISCUSSION TITLE
a couple of questions regarding firmware build

### DISCUSSION THREAD

**Discussion by phe78** [ZuluSCSI/ZuluSCSI-firmware#289] [Q&A]
Hi everyone, I succesfully completed the firmware build with platformIO, however I noticed to things that I'd lile possibly to clarify before uploading the code in my board : 
- PlatformIO complains about the ini file as follows : Warning! Ignore unknown configuration option `ldscript_bootloader`. Have you got the same warning and I can ignore it ? 
- the produced binary looks fine identifcal when comparing the top and bottom of the files in hex. However, the binary i generate is slightly longer (approx 700 bytes) than the one available for download on github. 

I ran the latest versions of Visual Code and PlatformIO, under W10. 
THanks in advance for your feedback
Philippe

**Comment by aperezbios** [comment]
@phe78 yes, that ldscript_bootloader warning is expected, and perfectly safe to ignore :) As for the slight difference in binary size, I don't know for sure what would cause that, but I personally don't find it unexpected. The firmware builds that are available for download are built via GitHub Actions/CI, on the GitHub end.
